docs/source/nn.rst

```diff
@@ -388,6 +372,28 @@ From the ``torch.nn.utils`` module
     spectral_norm
     remove_spectral_norm
 
+Utility functions to parametrize Tensors on existing Modules.
+Note that these functions can be used to parametrize a given Parameter
+or Buffer given a specific function that maps from an input space to the
+parametrized space. They are not parameterizations that would transform
```

Comment:
nit. perhaps "parameter space"?

Reply:
I meant that this is the space that is parametrized (or follow the constraints implied by the function).